server/netty/src/main/java/io/deephaven/server/netty/NettyConfig.java

```diff
@@ -0,0 +1,45 @@
+package io.deephaven.server.netty;
+
+import io.deephaven.annotations.BuildableStyle;
+import io.deephaven.server.config.ServerConfig;
+import org.immutables.value.Value.Default;
+import org.immutables.value.Value.Immutable;
+
+/**
+ * The netty server configuration.
+ */
+@Immutable
+@BuildableStyle
+public abstract class NettyConfig implements ServerConfig {
+
+    public static final int DEFAULT_SSL_PORT = 443;
+    public static final int DEFAULT_PLAINTEXT_PORT = 8080;
```

Comment:
why force 8080, why not the actual default plaintext port of 80?

if we allow port numbers <1024 (i.e. 443), we already require that we can run as root, so 80 should be fine?

Reply:
I chose it this way to leave the current default behavior the same and working out-of-the-box for localhost - no config -> plaintext + 8080. ssl without an explicit port -> ssl + 443.

I think when you are going through the act of ssl configuration, you'll be aware that you need privs to run at 443 (or, you'll know soon enough after startup), or you can do the small additional configuration for a port >= 1024.